relayer/relays/beacon/header/header.go

```diff
@@ -0,0 +1,262 @@
+package header
```

Comment:
This file has just been moved from `main.go` to `header.go`, no major new changes here.